apps/test/solutions/applab/ec_turtle.js

```diff
@@ -21,7 +21,7 @@ module.exports = {
         // room to add tests here
         assert($('#screen1'));
         assert($('#turtleImage'));
-        assert.equal($('#divApplab .screen').length, 1);
+        assert.equal($('#divApplab > .screen').length, 1);
```

Comment:
It's kind of like "the inspector" anti-pattern http://blog.james-carr.org/2006/11/03/tdd-anti-patterns/ but it's a bit of a stretch.